### DISCUSSION TITLE
Whether the finite element order is different in generating the preprocessing matrix and solving the equation？

### DISCUSSION THREAD

**Discussion by Moyiqi** [idaholab/moose#24029] [Q&A General] [answered]
Hello everyone! May I ask a question?

I have an idea to use linear finite element to generate preprocessing matrix M and quadratic element to solve it, but I don't know whether this idea can be realized in moose or petsc? How can I do that?

Lily

**Comment by lindsayad** [comment, marked as answer]
So you want your M and J to represent a first order basis and second order basis respectively? If that is what you're asking, this is not currently possible in MOOSE. We supply the same matrix for both M and J

**Reply by Moyiqi** [reply]
Thank you for your help!
Lily
